specificatie/openapi.yaml

```diff
@@ -182,12 +182,33 @@ components:
           $ref: '#/components/schemas/PersoonTypeEnum'
         vestigingsnummer:
           type: string
-    PersoonTypeEnum:
-      type: string
-      enum:
-        - "natuurlijk_persoon"
-        - "niet_natuurlijk_persoon"
-        - "vestiging"
+    KadastraleAanduiding:
+      type: object
+      properties:
+        gemeentecode:
+          description: Waar het kadaster de kadastrale gemeentenaam hanteert in de KadstraleAanduiding is in de WOZ de gemeentecode gebruikt.
+          type: string
+        sectie:
+          description: Sectie is een onderverdeling van de kadastrale gemeente, bedoeld om het werk van de meetdienst en de kadastrale kaarten overzichtelijk te houden. Per kadastrale gemeente zijn er max. 26x26 secties . Elke sectie heeft een of twee letters (bijv. "A", 'B', ....'AA', 'AB', ....'ZZ' . Alleen de sectieletter 'J' wordt niet gebruikt om verwarring (handgeschreven) te voorkomen met 'I'
+          type: string
+        perceelnummer:
+          description: Het perceelnummer dat een geheel perceel of een complex uniek identificeert binnen de sectie.
+          type: integer
+        deelperceelnummer:
+          type: string
+        appartementsindex:
+          description: Nummer dat het kadastraal object uniek identificeert als een appartementsrecht binnen het complex.
+          type: integer
+    KadastraalOnroerendeZaak:
+      type: object
+      properties:
+        identificatie:
+          type: string
+        aanduiding:
+          $ref: "#/components/schemas/KadastraleAanduiding"
+        aanduidingMetGemeentenaam:
+          description: De volledige kadastraleAanduiding waarbij alle delen zijn samengevoegd en waarbij de gemeentenaam is gebruikt.
```

Comment:
'... en waarbij de gemeentenaam is gebruikt.' --> '...en waarbij de gemeentenaam is gebruikt i.p.v. de gemeentecode.'